src/org/components/EventForm/steps/OrganizerStep.tsx

```diff
@@ -287,86 +283,43 @@ export const OrganizerStep = ({
-                            : `${contactPerson.first_name} ${contactPerson.last_name}`)
-                        }
-                      />
-                    </FormInputError>
-                  </InlineSection>
-                  <InlineSection>
-                    <Label htmlFor="propagation.contact_email">
-                      Kontaktní email
-                    </Label>
-                    <FormInputError>
-                      <input
-                        id="propagation.contact_email"
-                        type="email"
-                        {...register('propagation.contact_email')}
-                        placeholder={contactPerson?.email ?? undefined}
-                      />
-                    </FormInputError>
-                  </InlineSection>
-                  <InlineSection>
-                    <Label htmlFor="propagation.contact_phone">
-                      Kontaktní telefon
-                    </Label>
-                    <FormInputError>
-                      <input
-                        id="propagation.contact_phone"
-                        type="tel"
-                        {...register('propagation.contact_phone')}
-                        placeholder={contactPerson?.phone}
-                      />
-                    </FormInputError>
-                  </InlineSection>
-                </FormSubsection>
+              <FormSection header="Kontaktní údaje" required onWeb>
+                <InlineSection>
+                  <Label htmlFor="propagation.contact_name">
+                    Jméno kontaktní osoby
+                  </Label>
+                  <FormInputError>
+                    <input
+                      type="text"
+                      id="propagation.contact_name"
+                      {...register('propagation.contact_name', { required })}
+                    />
+                  </FormInputError>
+                </InlineSection>
+                <InlineSection>
+                  <Label htmlFor="propagation.contact_email">
+                    Kontaktní email
+                  </Label>
+                  <FormInputError>
+                    <input
+                      id="propagation.contact_email"
+                      type="email"
+                      {...register('propagation.contact_email')}
+                    />
+                  </FormInputError>
+                </InlineSection>
+                <InlineSection>
+                  <Label htmlFor="propagation.contact_phone">
+                    Kontaktní telefon
+                  </Label>
+                  <FormInputError>
+                    <input
+                      id="propagation.contact_phone"
+                      type="tel"
+                      {...register('propagation.contact_phone')}
+                    />
+                  </FormInputError>
+                </InlineSection>
```

Comment:
Validaci _'e-mail nebo telefon'_ můžeme přidat takto:

(Pokud je validace složitější, můžeme použít `yup` a mít validace v jednom místě.)

                 
 ```                
 <InlineSection>
                  <Label htmlFor="propagation.contact_email">
                    Kontaktní email
                  </Label>
                  <FormInputError>
                    <input
                      id="propagation.contact_email"
                      type="email"
                      {...register('propagation.contact_email', {
                        validate: value =>
                          value || watch('propagation.contact_phone')
                            ? true
                            : 'Vyplňte buď kontaktní email nebo telefon',
                      })}
                    />
                  </FormInputError>
                </InlineSection>
                <InlineSection>
                  <Label htmlFor="propagation.contact_phone">
                    Kontaktní telefon
                  </Label>
                  <FormInputError>
                    <input
                      id="propagation.contact_phone"
                      type="tel"
                      {...register('propagation.contact_phone', {
                        validate: value =>
                          value || watch('propagation.contact_email')
                            ? true
                            : 'Vyplňte buď kontaktní email nebo telefon',
                      })}
                    />
                  </FormInputError>
                </InlineSection>
```

Reply:
A do `useEffect` přidat 2 podmínky, aby se to triggerovalo během úpravy formuláře:

```      if (methods.formState.isSubmitted) {
        if (name === 'main_organizer') trigger('other_organizers')
        if (name === 'main_organizer' || name === 'other_organizers')
          trigger('propagation.organizers')
        if (name === 'propagation.contact_email')
          trigger('propagation.contact_phone')
        if (name === 'propagation.contact_phone')
          trigger('propagation.contact_email')
      
```

Reply:
@talitadzikzlasu přidáno, dík za vysvětlení

Reply:
👍🏻